### PR TITLE
BaSP-MR-103: Members Form

### PR DESCRIPTION
**Pull request by Ginoboca1**
Members Form Create
Members Form Edit

### CODE REVIEW

**Review by FrancoLelli — APPROVED** [review]

File: src/Components/Members/Form/MembersForm.js [hunk 1]
```diff
@@ -1,74 +1,107 @@
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
```
Comment (YhanahiRosenthal) [inline src/Components/Members/Form/MembersForm.js:1]: you should remove import component repeated

File: src/Components/Members/Form/form.module.css [hunk 2]
```diff
@@ -1,58 +1,65 @@
...
 }
 
-.input{
-  padding:5px;
-  border-radius:5px;
+.input {
+  padding: 5px;
+  border-radius: 5px;
   border: solid var(--Blue) 1px;
+} */
```
Comment (YhanahiRosenthal) [inline src/Components/Members/Form/form.module.css:29]: you should delete all unnecessary comments

File: src/Components/Layout/index.js [hunk 3]
```diff
@@ -31,6 +32,7 @@ function Layout() {
           <Route exact path="/classes" component={Classes} />
           <Route path="/classes/ClassForm/:id?" component={FormClasses} />
           <Route path="/members" component={Members} />
+          <Route path="/members/membersForm/:id?" component={MembersForm} />
```
Comment (FrancoLelli) [inline src/Components/Layout/index.js:35]:
```suggestion
          <Route path="/members/form/:id?" component={MembersForm} />
```

**Review by FrancoLelli — CHANGES_REQUESTED** [review]
Change path!!

**Review by FrancoLelli — APPROVED** [review]

**Comment by Barbimillan** [comment]
Gino, in the second test I found this bugs: 
Description: I click on the edit option and the title to the form says Add Member, and then the box modal to confirm also says Are you sure you want to add this member? Then when I confirm the edition appear a Box modal with member deleted! before the modal member edited

Steps:

Step 1 - Open home page.

Step 2 - Go to Member.

Step 3 - Click to edit member.

Step 4 - Complete the form.

Step 5 - Click submit

Expected result: Title Edit Member. When I submit a Box modal Are you sure you want to edit this member?, and then to click ok a box modal that says member edited successfully.

Actual result:  The form has the title Add member, the box model says Are you sure you want to add this member?, and then to click ok appear a box modal that says member deleted successfully!

**Pull request merged**